studio/schemas/documents/fleetCard.js

```diff
@@ -0,0 +1,22 @@
+export default {
+    name: 'fleetCard',
+    title: 'Fleet Card',
+    type: 'document',
+    fields: [
+        {
+            name: 'heading',
+            title: 'Heading',
+            type: 'string',
+        },
+        {
+            name: 'description',
+            title: 'Description',
+            type: 'text',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'mainImage',
+        },
+    ],
+};
```

Comment:
I don't think you'll need this file at all. You can get all of the data for the ui card element by querying for the individual fleet items.

Reply:
so should we include a description section on the individual fleet item for the card? @asasmith

Reply:
Yes. We can also reuse that field for seo description